packages/core/src/App/Components/Routes/binary-routes.jsx

```diff
@@ -15,13 +35,25 @@ const BinaryRoutes = observer(props => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [location]);
 
+    const title_TH = localize("Trader's Hub");
+    const title_TH_logged_out = localize('Deriv App');
+
     return (
         <React.Suspense fallback={<Loading />}>
             <Prompt when={prompt_when} message={promptFn} />
             <Switch>
-                {getRoutesConfig().map((route, idx) => (
-                    <RouteWithSubRoutes key={idx} {...route} {...props} />
-                ))}
+                {getRoutesConfig().map((route, idx) => {
+                    return <RouteWithSubRoutes key={idx} {...route} {...props} />;
+                })}
+                <RouteWithSubRoutes
```

Comment:
@aum-deriv You just moved Appstore, Wallets and Page 404 here from route-config, just because you need values from store? It looks bad as for me, because here you describe 3 additional routes, but we have routes-config for this... I don't understand why TH properties should be described here.